cli/cli/gateway.py

```diff
@@ -203,16 +202,14 @@ def get_jwt_creds(self, consumer_name: str) -> List[JwtCredential]:
         creds_data = resp.json()["data"]
         return [JwtCredential.from_json(d) for d in creds_data]
 
-    def print_jwt_cred(self, cred: JwtCredential, header=True):
-        if header:
-            click.secho(f"{'ALGORITHM':<15} {'SECRET':<40} {'ISS':<40}", bold=True)
-        click.secho(f"{cred.algorithm:<15} {cred.secret:<40} {cred.iss:<40}")
+    def print_jwt_creds(self, creds: list[JwtCredential], header=True):
+        click.secho(f"{'ALGORITHM':<15} {'SECRET':<40} {'ISS':<40}", bold=True)
```

Comment:
IMO printing the header even when there aren't any creds is useful to show an empty list. Without it it's not clear that anything has happened.